src/elfpy/wallet.py

```diff
@@ -113,13 +113,10 @@ def get_state(self, market: Market) -> dict:
 
         .. todo:: TODO: return a dataclass instead of dict to avoid having to check keys & the get_state_keys func
         """
-        lp_token_value = (
-            market.market_state.share_reserves
-            * market.market_state.share_price
-            * (self.lp_tokens / market.market_state.lp_reserves)
-            if self.lp_tokens > 0
-            else 0.0
-        )
+        share_of_pool = self.lp_tokens / market.market_state.lp_reserves if market.market_state.lp_reserves > 0 else 0.0
```

Comment:
I think it would be more clear to put this whole assignment in a loop, since it results in `lp_token_value` = 0. i.e.:
```
if market.market_state.lp_reserves > 0:
    # compute lp_token_value
else:
    lp_token_value = 0
```